mandatory/4-tax.js

```diff
@@ -17,7 +20,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(sales) {
+  return "£" + calculateSalesTax(sales) .toFixed(2);
```

Comment:
Smart, but this is doing quite a lot on one line. Adding an extra line with a variable - in the way you did in the function above - could make this more clear.